src/rdkafka_msgset_writer.c

```diff
@@ -931,9 +936,18 @@ static int
 rd_kafka_msgset_writer_compress_lz4 (rd_kafka_msgset_writer_t *msetw,
                                      rd_slice_t *slice, struct iovec *ciov) {
         rd_kafka_resp_err_t err;
+        int compLevel =
+                msetw->msetw_rktp->rktp_rkt->rkt_conf.compression_level;
+        if (compLevel>RD_KAFKA_COMPLEVEL_LZ4_MAX) {
+                compLevel = RD_KAFKA_COMPLEVEL_LZ4_MAX;
+        }
```

Comment:
`} else {`

Please see the style guide: https://github.com/edenhill/librdkafka/blob/master/CONTRIBUTING.md#follow-code-style

Reply:
a73440a